ui/app/AppLayouts/Wallet/views/RightTabBaseView.qml

```diff
@@ -34,7 +34,7 @@ FocusScope {
             networkConnectionStore: root.networkConnectionStore
         }
 
-        Column {
+        Item {
```

Comment:
This change will generate a different behaviour in terms of its usage further by not providing positioning anymore. Did you check that the intended behaviour is kept for the controls that use it?